calico/operations/operator-migration.mdx

```diff
@@ -54,10 +54,11 @@ Do not edit or delete any resources in the `kube-system` Namespace during the fo
 
 :::
 
-1. Install the Tigera $[prodname] operator and custom resource definitions.
+1. Install the Tigera operator and custom resource definitions.
 
    ```bash
    kubectl apply --server-side --force-conflicts -f $[manifestsUrl]/manifests/tigera-operator.yaml
+   kubectl apply --server-side --force-conflicts -f $[manifestsUrl]/manifests/tigera-operator.yaml
```

Comment:
The line above needs to be switched to `operator-crds.yaml`

Reply:
Good spot, thanks. Fixed.